### PR TITLE
Remove profanity and slurs

### PR DESCRIPTION
**Pull request by tristangosnell**

### CODE REVIEW

**Comment by bittner** [comment]
:+1:  That's a fun one!  :laughing:  :stuck_out_tongue:

File: xkcdpass/static/default.txt [hunk 1]
```diff
@@ -59159,8 +59079,7 @@ tiresomeness
 tiring
 tiro
 tissue
-tissue paper
-tit
+tissue pap
```
Comment (thiderman) [inline xkcdpass/static/default.txt]: ? :)

**Comment by thiderman** [comment]
I disagree with many of these. For example, `damnation` and `homoerotic` are neither slurs nor profane, and it is a stretch to even call them offensive.

What's the goal and intended value with this PR? The list added is just a copy of `cracklib-small`, one of the de facto standard files for wordlists in UNIX land. Why modify it? If you roll a password that contains something you don't want in a password, why not just roll another one?

**Comment by tristangosnell** [comment]
@thiderman  Agreed. Trust me, I'm no prude. Not trying to censor anyone. 

I'm using this library to generate passwords for reset emails for an internal service. I didn't come across any generated passwords in my testing that suggested the dictionary included profanity. Then a few awkward/hilarious emails from users later, I went through the default list and removed anything that I thought might even raise an eyebrow. Just hoping to save any future users from potential embarrassment or worse.

Perhaps adding an office-safe.txt is a better solution?

**Comment by burdges** [comment]
Anything biology related or dual use should stay.  If you cannot handle seeing the word breast, then that's really your own problem.  Any reclaimed/ex- slurs like gay should stay too.  Imho, removing them is only coddling the bigots who lost the fight over that term.  Imho, the mild curses should stay too, well mostly words like hell are dual use anyways. 

Arguably there is a case for removing the derivatives of chink, nigger, fag, and fuck as they're fairly heavy, not dual use, and not completely reclaimed by their respective communities.  I don't see any others though.

**Comment by tuxlifan** [comment]
From a security point of view the word list and choosing algorithm employed must be as unbiased as possible.
Therefore an additional office-safe.txt word list, together with with a warning in the README, or a filter script to run the original list through seems to me to be the best 'solution'.

**Comment by redacted** [comment]
(Apologies for the delay in replying to this, was overseas without reliable internet access).

My feeling is that adding a separate office-safe.txt to the repo is the best answer: it doesn't reduce the resulting password entropy in the default configuration, and it provides an easy alternative for situations like the ones Tristan encountered.

I actually came across such a list a while ago, created by a user of the software. It is heavily filtered to remove as much potentially offensive combinations as possible. I reached out and he gave me permission to include it in the repo: 

https://github.com/redacted/XKCD-password-generator/blob/master/contrib/office-safe.txt

I have also added a note and warning to the README. If this addresses the issue I will close the PR and leave default.txt alone.

**Comment by tristangosnell** [comment]
@redacted Thanks!

**Comment by redacted** [comment]
No worries, and thanks for spending time on this as well!

**Pull request closed without merging**